conans/client/conan_api.py

```diff
@@ -925,6 +925,10 @@ def remote_add(self, remote_name, url, verify_ssl=True, insert=None, force=None)
     def remote_remove(self, remote_name):
         return self.app.cache.registry.remove(remote_name)
 
+    @api_method
+    def remote_set_disabled(self, remote_name, state):
```

Comment:
Not sure about it, but maybe it is better `set_disabled_state` as it can enable remotes too... maybe there is a better name